### DISCUSSION TITLE
FilePond Field Type - alternative to dropzone

### DISCUSSION THREAD

**Discussion by tabacitu** [Laravel-Backpack/community-forum#295] [General] [answered]
# Feature Request

### What's the feature you think Backpack should have?

https://pqina.nl/filepond/

Similar to dropzone.

### Have you already implemented a prototype solution, for your own project?

No i have not 😀

### Do you see this as a core feature or an add-on?

Not sure yet.

**Comment by ziming** [comment]
Filepond is pretty nice. Used it in 2 of my projects so far. (The non admin pages)

**Comment by martijnb92** [comment, marked as answer]
Looks great, however the list of dependencies is getting huge. Maybe do some tree-shaking before creating another field type?

**Comment by tabacitu** [comment]
Thanks for the reality check @martijnb92 ! 🤣 You're right, of course 😄

**Comment by mitmelon** [comment]
Honestly i love File Pond... I dont just like dropzone... i dont know why? it doesnt work for me...

I think you should compile all the plugins into one file and add it to FilePond so everything becomes just only one distributions... Instead of uploading multiple plugins

**Comment by tabacitu** [comment]
No benefit in adding another library for the same thing. So let's close this.
